packages/types/src/payment-types.ts

```diff
@@ -83,18 +83,24 @@ export enum BALANCE_ERROR_CODE {
   VERSION_NOT_SUPPORTED,
 }
 
+export interface IPaymentNetworkBaseEvent {
+  name: EVENTS_NAMES;
+  timestamp?: number;
+}
+
 /** payment network event */
-export interface IPaymentNetworkEvent<TEventParameters> {
+export interface IPaymentNetworkEvent<TEventParameters> extends IPaymentNetworkBaseEvent {
   amount: string;
-  name: EVENTS_NAMES;
   parameters?: TEventParameters;
-  timestamp?: number;
 }
 
 /** payment network event names */
 export enum EVENTS_NAMES {
   PAYMENT = 'payment',
   REFUND = 'refund',
+  FROZEN_PAYMENT = 'frozenPayment',
```

Comment:
I'd rather see those 3 event names in their own enum instead of impacting an enum used by all...

Reply:
Thanks, I´ll fix it. :)

Reply:
```suggestion
}
/** escrow related payment network event names */
export enum ESCROW_EVENTS_NAMES {
  FROZEN_PAYMENT = 'frozenPayment',
  INITIATED_EMERGENCY_CLAIM = 'initiatedEmergencyClaim',
  REVERTED_EMERGENCY_CLAIM = 'revertedEmergencyClaim',
}
```

Reply:
@benjlevesque - Something like this? Can you review it?

Reply:
this seems OK, not too much impact?